src/reducers/app.js

```diff
@@ -20,11 +20,11 @@ const initialState = {
     fr: "Français",
     es: "Español" ,
   },
-  activeLanguage: 'en',
-  account: {
-    key: undefined,
-    address: undefined,
-    isLoggedIn: false,
+  activeLanguage: (Lockr.get("language") !== undefined ? Lockr.get("language") : 'en'),
```

Comment:
`Locker.get` has a second parameter for a default value, this can be simplified  to `Lockr.get("language", "en")`